docs/querying/api.md

```diff
@@ -268,6 +268,12 @@ GET /api/v1/labels
 POST /api/v1/labels
 ```
 
+URL query parameters:
+
+- `start=<rfc3339 | unix_timestamp>`: Start timestamp. Optional.
```

Comment:
I forgot about would be nice to mention that labels can be OUTSIDE of given ranges (:

Reply:
As this is just a hint